pt-BR/mail/chrome/messenger/msgHdrViewOverlay.dtd

```diff
@@ -35,34 +35,34 @@
 
  ***** END LICENSE BLOCK ***** -->
 
-<!ENTITY pbToField.label "To:">
-<!ENTITY pbFromField.label "From:">
-<!ENTITY pbSenderField.label "Sender:">
-<!ENTITY pbOrganizationField.label "Organization:">
-<!ENTITY pbReplyToField.label "Reply-To:">
+<!ENTITY pbToField.label "Para:">
+<!ENTITY pbFromField.label "De:">
+<!ENTITY pbSenderField.label "Remetente:">
+<!ENTITY pbOrganizationField.label "Organização:">
+<!ENTITY pbReplyToField.label "Responder para:">
 
-<!ENTITY subjectField.label "Subject: ">
+<!ENTITY subjectField.label "Assunto: ">
 <!--# LOCALIZATION NOTE (ccField.label): DONT_TRANSLATE -->
 <!ENTITY pbCcField.label "Cc:">
-<!ENTITY pbBccField.label "Bcc:">
-<!ENTITY pbNewsgroupsField.label "Newsgroups:">
-<!ENTITY pbFollowupToField.label "Followup-To:">
+<!ENTITY pbBccField.label "Cco:">
+<!ENTITY pbNewsgroupsField.label "Grupos de mensagens:">
+<!ENTITY pbFollowupToField.label "Encaminhar para:">
 <!ENTITY pbLinkField.label "Link:">
-<!ENTITY pbTagsHdr.label "Topics:">
-<!ENTITY pbDateField.label "Date:">
-<!ENTITY pbUserAgentField.label "User-Agent:">
-<!ENTITY pbReferencesField.label "References:">
-<!ENTITY pbMessageIdField.label "Message-ID:">
-<!ENTITY pbInReplyToField.label "In-Reply-To:">
-<!ENTITY pbOriginalWebsite.label "Website:">
-<!ENTITY pbParticipantsField.label "Participants:">
-<!ENTITY saveAll.label "Save All..">
+<!ENTITY pbTagsHdr.label "Tópicos:">
+<!ENTITY pbDateField.label "Data:">
+<!ENTITY pbUserAgentField.label "Agente do usuário:">
```

Comment:
I disagree. This is the "User-Agent" field shown on the header of messages, and it refers to the email client used by the user.